### ISSUE TITLE
[BUG] Builder is broken in version: 4.8.0-rc.11

### ISSUE THREAD

**Issue by gregbown** [formio/formio.js#2245]
- Hosting type
  - [x] Form.io
  - [ x] Local deployment
    - Version: 4.8.0-rc.11
- Formio.js version: 4.8.0-rc.11
- Frontend framework: NodeJS/Express
- Browser: Chrome 
- Browser version:  79.0.3945.88 

## Steps to Reproduce
1. Drag any component from builder menu to layout
2. The component is not reflected in the change event data and is not rendered in preview
3. Rolling back to Formio.js version 4.8.0-rc.10 corrects the problem

## Expected behavior
Form builder should reflect added component in data
## Observed behavior
Form builder is broken/unusable because you cannot build forms
## Example
Version =<4.8.0-rc.10
```
builder change event 
display: "wizard"
components: Array(1)
	0:
		title: "Page 1"
		label: "Page 1"
		type: "panel"
		key: "page1"
		components: Array(1)
			0:
				label: "Text Field"
				labelPosition: "top"
				placeholder: ""
				description: ""
				tooltip: ""
				prefix: ""
				suffix: ""
				widget: {type: "input"}
```
Version >4.8.0-rc.10
```
builder change event 
display: "wizard"
components: Array(1)
	0:
		title: "Page 1"
		label: "Page 1"
		type: "panel"
		key: "page1"
		components: Array(0)
```

**Comment by jhen1422** [comment]
@gregbown Thank you for the report. We are aware of this issue. Please use the previous version while we get this resolved.

**Comment by travist** [comment]
Fixed with 4.8.0-rc.12
